proposals/exception-handling/Exceptions.md

```diff
@@ -258,6 +258,18 @@ end
 The `rethrow` here references `try $l2`, but the `rethrow` is not within its
 `catch` block.
 
+Also, targetting a label of `catch`-less `try` or `try`-`delegate` is also a
+validation failure, because `rethrow` is not within a `catch` or `catch_all`.
```

Comment:
This part is confusing as well: even if there was a surrounding catch somewhere it would still be invalid. Whether a catch _exists_ is irrelevant, it only matters whether the rethrow _references_ a catch.

Reply:
Deleted.